lib/cartopy/crs.py

```diff
@@ -1778,6 +1778,75 @@ def y_limits(self):
         return self._y_limits
 
 
+class Sinusoidal(Projection):
+    """
+    A Sinusoidal projection
+
+    This projection provides generally accurate sizes, but angles are distorted
+    toward the bounds.
+    """
+
+    def __init__(self, central_longitude=0.0, false_easting=0, false_northing=0,
+            a=6371007.181, b=6371007.181, globe=None):
```

Comment:
I added it in because it's in the proj4 string at http://spatialreference.org/ref/sr-org/6842/proj4/ (and it usually isn't). Should I take it out?